First_Training_Model.py

```diff
@@ -0,0 +1,49 @@
+from fastai.vision.all import *
+import os
+from pathlib import Path
+
+
+import wandb
+
+run = wandb.init()
+artifact = run.use_artifact(
+    "arcslaboratory/wandering-random-2K+/07-26_wandering_10Trials_randomized:v0",
+    type="dataset",
+)
+artifact_dir = artifact.download()
+
+
+image_dir = os.path.join(artifact_dir, "data")
+
+# Use the `get_image_files` function to get image files from the image_dir
+files = get_image_files(image_dir)
+
+
+def label_func(f):
```

Comment:
This function needs some documentation. It would be good to put some example filenames in the documentation.